keps/sig-apps/3335-statefulset-slice/README.md

```diff
@@ -435,10 +437,11 @@ This can inform certain test coverage improvements that we want to do before
 extending the production code to implement this enhancement.
 -->
 
-*   `pkg/controller/statefulset/stateful_set_control_test.go - Tests that a StatefulSet slice can be created from specified starting ordinal`
-*   `pkg/apis/apps/v1/defaults_test.go - Tests defaults for new fields added to StatefulSet`
+*   `pkg/apis/apps/validation/validation_test.go` - Tests that the .spec.ordinals.start value is properly validated.
```

Comment:
There is no need to describe what unit tests do - please use the template mentioned above:
```
<package>: <date> - <current test coverage>
```

Reply:
Done